sdk/go/pulumi-language-go/main.go

```diff
@@ -331,31 +332,54 @@ func (host *goLanguageHost) GetRequiredPlugins(ctx context.Context,
 	}, nil
 }
 
-func execProgramCmd(cmd *exec.Cmd, env []string) error {
+func runCmdStatus(cmd *exec.Cmd, env []string) (int, error) {
 	cmd.Env = env
 	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
 
 	err := cmd.Run()
```

Comment:
This err handling is incomplete. It only catches subclasses of exec.ExitError. Is that ok?